packages/ember-auto-import/ts/tests/analyzer-test.ts

```diff
@@ -487,4 +487,17 @@ Qmodule('analyzer-deserialize', function () {
     let result = await deserialize(source([`stuff stuff stuff ${meta.slice(0, -2)}`, meta.slice(-2)]));
     assert.deepEqual(result, sampleData());
   });
+
+  test('false end marker at end of chunk', async function (assert) {
+    const meta = serialize(sampleData());
+    assert.ok(
+      meta.slice(MARKER.length, -MARKER.length).indexOf(MARKER[0]) > -1,
+      'serialized sample data must contain first character of MARKER somewhere between boundary markers for test to have meaning'
+    );
```

Comment:
This assertion isn't really checking the result of running code -- its just checking that our sample data is sufficient for this test case.  If `MARKER` or the sample data were changed in the future, such that the first character of `MARKER` didn't show up somewhere in the serialized data, this test would become invalid.